test/keystore/InMemoryKeystore.test.ts

```diff
@@ -472,6 +475,39 @@ describe('InMemoryKeystore', () => {
       }
     })
 
+    it('uses deterministic topic', async () => {
+      const recipient = SignedPublicKeyBundle.fromLegacyBundle(
+        bobKeys.getPublicKeyBundle()
+      )
+      const baseTime = new Date()
+      const timestamps = Array.from(
+        { length: 25 },
+        (_, i) => new Date(baseTime.getTime() + i)
+      )
+
+      // Shuffle the order they go into the store
+      const shuffled = [...timestamps].sort(() => Math.random() - 0.5)
+
+      await Promise.all(
+        shuffled.map(async (createdAt) => {
+          return aliceKeystore.createInvite({
+            recipient,
+            createdNs: dateToNs(createdAt),
+            context: undefined,
+          })
+        })
+      )
+
+      const convos = await aliceKeystore.getV2Conversations()
+
+      // Ensure all the convos have the same topic
+      expect(
```

Comment:
I think `convos` above might return just one conversation (deduplicated off topic). A more simplistic/readable test case could be something like:
```
      // Go through CreateInviteResponses and ensure they all have the same topic
      const firstResponse = responses[0]
      const topicName = firstResponse.conversation!.topic

      expect(
        responses.filter((value, index, array) => {
          return value.conversation!.topic === topicName
        })
      ).toHaveLength(25)
```

Reply:
I tried to think of a clean way to test `topicKey` is the same, but it's harder since the keyMaterial is not surfaced in the response